plugins/modules/template_workflow_manager.py

```diff
@@ -1498,7 +1515,7 @@ def get_containing_templates(self, containing_templates):
 
             project_name = item.get("project_name")
             if project_name is None:
-                self.msg = "project_name is mandatory under containing templates"
+                self.msg = "project_name is required under containing templates"
```

Comment:
The parameter 'project_name' is required under 'containing_templates' but not provided.